src/main.rs

```diff
@@ -269,14 +407,27 @@ async fn main() {
 
     let shard_manager = client.shard_manager.clone();
 
-    let runtime = tokio::runtime::Runtime::new().unwrap();
+    let winner_runtime = tokio::runtime::Runtime::new().unwrap();
+    let giveaway_runtime = tokio::runtime::Runtime::new().unwrap();
     let mut scheduler = Scheduler::with_tz(chrono::Local);
 
-    let db = client.get_db().await;
-    let http = client.cache_and_http.http.clone();
-
+    let winner_http = client.cache_and_http.http.clone();
+    let winner_db = client.get_db().await;
     scheduler.every(1.day()).at("00:00").run(move || {
-        runtime.block_on(display_winner(http.to_owned(), db.to_owned(), 1));
+        winner_runtime.block_on(display_winner(
+            winner_http.to_owned(),
+            winner_db.to_owned(),
+            1,
+        ));
+    });
+
+    let giveaway_http = client.cache_and_http.http.clone();
+    let giveaway_db = client.get_db().await;
+    scheduler.every(5.seconds()).run(move || {
```

Comment:
This is called way too often, maybe 10 secs, 30secs, or even minute?, we dont need that accurate giveaway times